build.gradle

```diff
@@ -216,6 +215,24 @@ publishing {
     }
 }
 
+jar {
+    manifest {
+        attributes(
+                'Implementation-Title': 'Firebolt JDBC driver', // This value must be equal to one defined in VersionUtil
+                'Implementation-Vendor': 'Firebolt Analytics',
+                'Implementation-Version': version,
```

Comment:
Will this work accounting that you've removed `version` variable?

Reply:
It is not removed but rather moved to gradle.properties